ui/imports/shared/panels/NoImageUploadedPanel.qml

```diff
@@ -0,0 +1,51 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import utils 1.0
+import shared.panels 1.0
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+/*!
+  /brief Image icon and ulopad text hints for banner and logo
+ */
+Item {
+    id: root
+
+    implicitWidth: mainLayout.implicitWidth
+    implicitHeight: mainLayout.implicitHeight
+
+    property bool showARHint: false
+
+    ColumnLayout {
+        id: mainLayout
+
+        SVGImage {
```

Comment:
can we avoid using this SVGIMage it should eventually be moved out completely. This is present because of the partial moving of stuff to StatusQ.